src/helpers/strings.ts

```diff
@@ -6,6 +6,8 @@
 
 /* tslint:disable:variable-name */
 export class Strings {
+    static ViewYourPinnedQuery: string = "View your pinned work item query.";
```

Comment:
It'd be great if you could change "...work item query" to "...work item query results."